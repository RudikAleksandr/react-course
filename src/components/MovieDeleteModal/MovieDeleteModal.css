
.modalDeleteMovie {
  backdrop-filter: blur(10px);
}

.modalDeleteMovie .modal-content {
  color: white;
  margin: auto;
  font-size: 1.2rem;
  padding: 40px 40px;
  padding-bottom: 40px;
  background-color: #232323;
  width: 75%;
}

.modalDeleteMovie .modal-header {
  padding-bottom: 0;
}

.modalDeleteMovie .modal-header h2 {
  color: white;
  text-transform: uppercase;
  font-weight: 300;
}

.modalDeleteMovie .modal-header,
.modalDeleteMovie .modal-footer {
  border: none;
}

.modalDeleteMovie .close {
  position: absolute;
  right: 22px;
  top: 10px;
}

.modalDeleteMovie .close,
.modalDeleteMovie .close:hover {
  font-size: 2.4rem;
  color: white;
}
