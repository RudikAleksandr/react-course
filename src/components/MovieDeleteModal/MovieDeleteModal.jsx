import React from 'react';
import { Modal, Button } from 'react-bootstrap';
import './MovieDeleteModal.css';

const MovieDeleteModal = ({ onHide, onConfirm }) => (
  <Modal
    centered
    size="lg"
    show={true}
    className="modalDeleteMovie"
    onHide={onHide}>
    <Modal.Header closeButton>
      <h2>delete movie</h2>
    </Modal.Header>
    <Modal.Body>
      <span>Are you sure want to delete this movie?</span>
    </Modal.Body>
    <Modal.Footer>
      <Button onClick={onConfirm}>Confirm</Button>
    </Modal.Footer>
  </Modal>
);

export default MovieDeleteModal;
