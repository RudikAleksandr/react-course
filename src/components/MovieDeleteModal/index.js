import MovieDeleteModal from './MovieDeleteModal.jsx';

export default MovieDeleteModal;
