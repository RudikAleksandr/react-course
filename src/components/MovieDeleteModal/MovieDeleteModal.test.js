
import { render } from '@testing-library/react';
import MovieDeleteModal from './MovieDeleteModal';

describe('MovieDeleteModal component', () => {
  test('render MovieDeleteModal snapshot', () => {
    const component = <MovieDeleteModal onHide={jest.fn()} onConfirm={jest.fn()} />
    const { asFragment } = render(component);

    expect(asFragment(component)).toMatchSnapshot();
  });
});