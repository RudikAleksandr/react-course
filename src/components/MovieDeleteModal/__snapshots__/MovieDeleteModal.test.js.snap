// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDeleteModal component render MovieDeleteModal snapshot 1`] = `<DocumentFragment />`;
